### PR TITLE
add contribex TLs to Peribolos OWNERS

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
Add SIG Contributor Experience Technical Leads to the Peribolos OWNERS file as approvers.

/sig contributor-experience
/assign @MadhavJivrajani @nikhita

### CODE REVIEW

File: prow/cmd/peribolos/OWNERS [hunk 1]
```diff
@@ -4,5 +4,7 @@ reviewers:
 - cjwagner
 approvers:
 - cjwagner
+- MadhavJivrajani
+- Priyankasaggu11929
```
Comment (MadhavJivrajani) [inline prow/cmd/peribolos/OWNERS:8]: We can replace this with an alias: https://github.com/kubernetes/test-infra/blob/9fd28d825479145276bcd5f64b16b0c49231971d/OWNERS_ALIASES#L68C3-L68C46

**Review by MadhavJivrajani — CHANGES_REQUESTED** [review]

File: prow/cmd/peribolos/OWNERS (hunk 1 above)
Comment (nikhita) [inline prow/cmd/peribolos/OWNERS:8]: Please also add me and @cblecker here :)

Comment (nikhita) [reply prow/cmd/peribolos/OWNERS]: @Priyankasaggu11929 I like Madhav's suggestion. Let's use this alias.

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

**Comment by Priyankasaggu11929** [comment]
(sorry for the enormous amounts of notification :sweat: )

I got real confused with the alphabetical order :laughing:

File: prow/cmd/peribolos/OWNERS (hunk 1 above)
Comment (Priyankasaggu11929) [reply prow/cmd/peribolos/OWNERS]: updated. Thanks! :pray:

Comment (Priyankasaggu11929) [reply prow/cmd/peribolos/OWNERS]: updated. Thanks! :pray:

**Review by MadhavJivrajani — APPROVED** [review]

**Review by palnabarun — APPROVED** [review]
/lgtm

**Comment by MadhavJivrajani** [comment]
@Priyankasaggu11929 let's add in the alias to reviewers too?

**Comment by Priyankasaggu11929** [comment]
> let's add in the alias to reviewers too?

@MadhavJivrajani, updated.

File: prow/cmd/peribolos/OWNERS [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 reviewers:
 - cjwagner
+- sig-contributor-experience-technical-leads
```
Comment (BenTheElder) [inline prow/cmd/peribolos/OWNERS:5]: We should really be adding people that have worked on this, which happens to be the current contribex leads.

but granting people sudden broad /approve opaquely by way of updating an alias like this is an anti-pattern I will continue to discourage, I don't think we should have ever added the feature

**Pull request merged**